train-the-trainers/black-hat-bash-back/basic-phishing-website/Vagrantfile

```diff
@@ -9,6 +9,9 @@
 Vagrant.configure("2") do |config|
   config.vm.box = "ubuntu/bionic64"
   config.vm.network "public_network"
+  config.vm.provider :virtualbox do |vb|
```

Comment:
This is using the [symbol](http://rubylearning.com/satishtalim/ruby_symbols.html) `:virtualbox`, but [Vagrant docs always use a string (`"virtualbox"`)](https://www.vagrantup.com/docs/providers/configuration.html#provider-configuration) for the argument to the `vm.provider` method call. Lets match the official Vagrant style here and use a string instead of a symbol, [like we do for the "Securing a shell account" practice lab](https://github.com/AnarchoTechNYC/meta/blob/b93df8a29d0986d8f5fb60b86f444d6277bd907d/train-the-trainers/practice-labs/securing-a-shell-account-on-a-shared-server/Vagrantfile#L5) as well.

Reply:
Got it; I went ahead and changed this, and while I was at it I changed `|vb|` to `|v|` since that is also how we had it in the "Securing a shell account" file and elsewhere.